server/src/test/java/io/crate/integrationtests/GroupByAggregateTest.java

```diff
@@ -673,9 +674,9 @@ public void testGroupByUnknownResultColumn() throws Exception {
     @Test
     public void testGroupByUnknownGroupByColumn() throws Exception {
         this.setup.groupBySetup();
-        Assertions.assertThrows(Exception.class,
-                                () -> execute("select max(birthdate) from characters group by details_ignored['lol']",
-                                              "Cannot GROUP BY type: undefined"));
+        assertThrows(() -> execute("select max(age) from characters group by details_ignored['lol']"),
+                     isSQLError(is("Using column type ignored 'details_ignored['lol']' for grouping or aggregations is not allowed"),
+                                INTERNAL_ERROR, BAD_REQUEST, 4000));
```

Comment:
psql returns a `42703` `undefined_column` in the case of an unknown column but this is not exactly the same. We mentioned in the docs that the type object ignored cannot not be used for grouping. https://crate.io/docs/crate/reference/en/4.2/general/ddl/data-types.html#ignored